src/components/empty-data/empty-data-illustration.component.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+export const EmptyDataIllustration = ({ width = "61", height = "59" }) => {
```

Comment:
Shouldn't you be using this? https://github.com/openmrs/openmrs-esm-patient-chart/blob/master/packages/esm-patient-common-lib/src/empty-state/empty-data-illustration.component.tsx

Reply:
I got it from there

Reply:
Should you duplicate it?

Reply:
Is there a way I can use this without duplicating it?

Reply:
How about just importing it from any of the esms that already have it?

Reply:
yeah I was able to import it